### PR TITLE
Remove storybook from web code

### PR DESCRIPTION
**Pull request by nghialv**
**What this PR does / why we need it**:

This is one of the steps to simplify our development process and codebase.

This PR removes all storybook files since they have become quite outdated compared to our recent UI changes and we don't use them much in our development process.

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/app/web/package.json [hunk 1]
```diff
@@ -11,8 +11,6 @@
     "test:coverage": "yarn test --coverage",
     "lint": "eslint --ext .ts,.tsx ./src",
     "lint:fix": "eslint --fix --ext .ts,.tsx ./src",
-    "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook",
```
Comment (khanhtc1202) [inline pkg/app/web/package.json:15]: should we remove storybook-related dependencies as well? this file at line L22-24

Comment (nghialv) [reply pkg/app/web/package.json:15]: Nice catch!
Tbh, not sure why but the build does not pass when I remove them. So weird!
Let me deal with them in another PR.

Comment (khanhtc1202) [reply pkg/app/web/package.json:15]: Sure 👍

**Comment by khanhtc1202** [comment]
Thanks Storybook for your serve
/approve

**Pull request merged**